include/boost/geometry/formulas/area_formulas.hpp

```diff
@@ -354,43 +363,44 @@ class area_formulas
     static inline CT spherical(PointOfSegment const& p1,
                                PointOfSegment const& p2)
     {
+        CT const pi = math::pi<CT>();
+
         CT excess;
 
-        if (LongSegment) // not for segments parallel to equator
+        CT const lon1r = get_as_radian<0>(p1);
+        CT const lat1r = get_as_radian<1>(p1);
+        CT const lon2r = get_as_radian<0>(p2);
+        CT const lat2r = get_as_radian<1>(p2);
+
+        CT lon12r = lon1r - lon2r;
```

Comment:
I'm aware that this naming was used before but I'm curious about the convention. Shouldn't this variable be called `lon21r` since it's an angle from `lon2r` to `lon1r`?

Azimuth between points of coordinates `lon1, lat1` and `lon2, lat2` is called `a12` in the code. Other variables referred to the azimuth e.g. in direct and inverse formulas are also called this way. Basically the order of numbers defines the direction.

Here the order of numbers defines the subtraction. This is also fine but I'm wondering whether or not it may be confusing.

Reply:
Good point! The wikipedia https://en.wikipedia.org/wiki/Geodesics_on_an_ellipsoid is also using what you propose (i.e. $\lambda_{12}=\lambda_2-\lambda_1$)
In my case since only the absolute value matters I change the order of the subtraction (following the convention you propose). That also simplifies the call to `trapezoidal_formula()` below.